test/functional/legacy/045_folding_spec.lua

```diff
@@ -33,24 +37,89 @@ describe('folding', function()
     execute('call append("$", getline("."))')
     feed('zc')
     execute('call append("$", getline(foldclosed(".")))')
-    -- Test folding with markers.
-    execute('set fdm=marker fdl=1 fdc=3')
-    execute('/^5')
-    execute('call append("$", "marker " . foldlevel("."))')
+
+    expect([[
+      manual 1 aa
+      -1
+      3 cc
+      1 aa]], true)
```

Comment:
The reason that the screen.lua expect() method has "any" parameter is because screen tests are unpredictable. Buffer contents should be predictable. Also you could use the VimL search() function or the Lua string.find() function on buffer contents. Neither of those would work when checking screen time-sensitive screen redraws.

So I think we should not add `any` paramter to helpers.expect().